webdriver-spec.html

```diff
@@ -2261,6 +2273,14 @@
       <p>Otherwise, set the "<code>proxy</code>" entry
        in <var>matched capabilities</var> to <var>capability value</var>.
 
+     <dt>"<code>setWindowRect</code>"
+     <dd><p>If <var>capability value</var> is not a boolean
```

Comment:
No you are understanding it correctly.

But if I send through
```
{
  "setWindowRect": "I made a mistake and specified a string instead of a boolean"
}
```
or
```
{
  "setWindowRect": False
}
```

The driver is just going to overwrite it and not going to care about what is in it. If some sends through 
```
{
  "setWindowRect": false
}
```

We can't guarantee that we can't manipulate the page. If it's set to false but some clever clogs might have buttons to go fullscreen and we can't block that.